security/docker/Dockerfile.node-agent-k8s

```diff
@@ -1,4 +1,6 @@
 # TODO(incfly): change to smaller base image before release.
 FROM ubuntu:16.04
 ADD node_agent_k8s /
+ADD nodeagent-sds-cert.pem /
```

Comment:
probably it's better putting binary/cert under some subfolder instead of root ?

Reply:
Done, `/etc/istio`.